_posts/2021-01-08-status-update.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: "2021-01-08: IAC Team Status Update"
+author: pmcmaw
+categories:
+  - team
+  - status
+tags:
+  - community
+  - releases
+  - dsc
+  - windows
+  - cloud-ci
+  - puppet-module-gem
+  - rubocop
+  - infrastructure
+  - networking
+  - puppetize
+  - triage
+---
+
+## Happy New Year!
+
+On behalf whole IAC team we would like to take the opportunity to wish all readers a Happy New Year! We hope you all had a great Christmas. The team took the opportunity enjoy the holidays, recharge our batteries and are now ready to tackle 2021 head on.
+
+![2021]({% link /assets/2020-01-08-status-update/2021.jpg %})
+
+## Well Hello TP!
+
+TP Honey has returned to the IAC team. [TP][TP] has been with Puppet for quite a while and has worked on multiple teams. This week we see him return to the IAC team to lend some expertise. [TP][TP] specialises in all things modules and has been a leader in the development of [litmus][litmus]. We are so delighted to have him onboard with us and you will see him around a lot more. You may recognise [TP][TP] as he has done some pretty cool Puppetize talks. Check out one of my favourites called [litmus_and_pdk][Litmus and PDK].
```

Comment:
```suggestion
TP Honey has returned to the IAC team. [TP][TP] has been with Puppet for quite a while and has worked on multiple teams. This week we see him return to the IAC team to lend some expertise. [TP][TP] specialises in all things modules and has been a leader in the development of [litmus][litmus]. We are so delighted to have him onboard with us and you will see him around a lot more. You may recognise [TP][TP] as he has done some pretty cool Puppetize talks. Check out one of my favourites called [Litmus and PDK][litmus_and_pdk].
```

Reply:
seems like a few other links also are suffering from this - please do a manual build locally and review all links before merging!

Reply:
thanks for that! Fixed them but will build locally to make sure I dont miss anything :-)